solutions/USACO jan19-guess-the-animal.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: usaco-839
+source : USACO Bronze 2019 January
+title: Guess The Animal
+author: Ethan Wu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_guess_bronze_jan19.html)
+<LanguageSection>
+<JavaSection>
+
+## Implementation
```

Comment:
Put the Implementation heading outside the LanguageSection and add the time complexity.

Reply:
Err... I don't think we need the time complexity for bronze problems... Putting time complexity is optional.

Reply:
Oh, sorry, I didn't know that. @HarambeTheGiraffe, add the time complexity if you want, then.